### PR TITLE
Add 1.2.4 entry.

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/text/news.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # News
 
+## 1.2.4: 2017-04-24
+
+### Improvements
+
+  * groonga-query-log-run-regression-test: Added `--old-groonga-option`
+    and `--new-groonga-option` option.
```
Comment (kou) [inline doc/text/news.md]: `options`.

File: doc/text/news.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+    Add an additional old or new groonga option.
```
Comment (kou) [inline doc/text/news.md]: `They add`

File: doc/text/news.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+    You can specify this option multiple times to specify multiple groonga
```
Comment (kou) [inline doc/text/news.md]: `these options`

File: doc/text/news.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+    options.
+
+  * groonga-query-log-run-regression-test: Added verify persistent cache.
```
Comment (kou) [inline doc/text/news.md]: `a persistent cache verification feature`.

File: doc/text/news.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+    When put `--old-groonga-option` or `--new-groonga-option` option,
```
Comment (kou) [inline doc/text/news.md]: `It verifies cache_hit_rate in status command results automatically when you specify --old-groonga-option=--cache-base-path or --new-groonga-option=--cache-base-path.`

**Comment by komainu8** [comment]
@kou Thank you very much for your comments. I've fixed it.

**Comment by kou** [comment]
Merged.

**Pull request merged**